deployments/helm/dra-example-driver/templates/kubeletplugin.yaml

```diff
@@ -6,10 +6,12 @@ metadata:
   namespace: {{ include "dra-example-driver.namespace" . }}
   labels:
     {{- include "dra-example-driver.labels" . | nindent 4 }}
+    app.kubernetes.io/component: kubeletplugin
```

Comment:
This label is used to distinguish the kubeletplugin from the webhook so selectors don't get confused.